src/ssl.c

```diff
@@ -26567,7 +26567,7 @@ int wolfSSL_X509_VERIFY_PARAM_set1_ip_asc(WOLFSSL_X509_VERIFY_PARAM *param,
             param->ipasc[0] = '\0';
         }
         else {
-            XSTRNCPY(param->ipasc, ipasc, WOLFSSL_MAX_IPSTR-1);
+            XSTRNCPY(param->ipasc, ipasc, WOLFSSL_MAX_IPSTR);
```

Comment:
@SparkiDev Was this intentional? Looks like gcc-10 flags it and @douzzer is suggesting adding back the `-1`.